dev/plugins/hu.elte.txtuml.xtxtuml.ui/src/hu/elte/txtuml/xtxtuml/ui/highlighting/XtxtUMLHighlightingConfiguration.xtend

```diff
@@ -36,4 +39,11 @@ class XtxtUMLHighlightingConfiguration extends XbaseHighlightingConfiguration {
 		return textStyle;
 	}
 
+	def enumerationLiteral() {
+		val textStyle = defaultTextStyle.copy;
+		textStyle.color = new RGB(0, 0, 192);
+		textStyle.style = SWT.ITALIC;
```

Comment:
No, because Java constants (static and final fields) are not bold in the Xtext editor by default (they are only bold in the Java editor).